presentation/src/main/res/values/dimens.xml

```diff
@@ -7,6 +7,13 @@
     <dimen name="margin_cardview">8dp</dimen>
     <dimen name="elevation">6dp</dimen>
     <dimen name="corner_radius">8dp</dimen>
+    <dimen name="sp_12">12sp</dimen>
+
+    <dimen name="dp_0">0dp</dimen>
+    <dimen name="dp_4">4dp</dimen>
+    <dimen name="dp_8">8dp</dimen>
+    <dimen name="dp_16">16dp</dimen>
+    <dimen name="dp_150">150dp</dimen>
```

Comment:
This is not a good name to name resources, please check this article: https://jeroenmols.com/blog/2016/03/07/resourcenaming/

Reply:
@recaldev The idea is that the dimensions that are meaningful made references to those base dimensions. Can we discuss it at the meeting?

Reply:
Sure, because this naming is very odd, there are better names for the base dimens.